### ISSUE TITLE
TypeError thrown when entering certain classes

### ISSUE THREAD

**Issue by BenAlderfer** [ComputerScienceHouse/schedulemaker#207]
Verified its working with other majors but I can consistently reproduce the bug with that class

![image](https://user-images.githubusercontent.com/9043915/64481081-dfbb1e00-d1a2-11e9-975e-a27d41ff7624.png)

**Comment by mxmeinhold** [comment]
Looking at SIS, I don’t see anything matching FIN 120. There is a FINC 120, which is the closest match I see.

**Comment by BenAlderfer** [comment]
Right, its an invalid department code. However, when I enter a non-existent class like "csci 102", it says "No courses match". This should probably be handled in a similar way rather than throwing an error
